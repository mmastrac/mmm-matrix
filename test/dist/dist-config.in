export INPUT_INPUT="
- os: linux
  \$if: config.json.expensive == false
- os: mac
  \$if: config.json.expensive == true
- os: windows
  \$if: config.json.expensive == true
"
export INPUT_CONFIG="
json: {
  "expensive": true
}
"
