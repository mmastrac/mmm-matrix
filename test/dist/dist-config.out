::group::Input matrix
- os: linux
  $if: config.json.expensive == false
- os: mac
  $if: config.json.expensive == true
- os: windows
  $if: config.json.expensive == true

::endgroup::
::group::Config object
json:
  expensive: true

::endgroup::
::group::Generated matrix
- os: mac
- os: windows

::endgroup::

::set-output name=matrix::[{"os":"mac"},{"os":"windows"}]
Success.
